slasher/detection/service.go

```diff
@@ -196,6 +196,11 @@ func (ds *Service) detectHistoricalChainData(ctx context.Context) {
 				}
 			}
 			ds.submitAttesterSlashings(ctx, slashings)
+
+			// update highest attestation
+			if err := ds.UpdateHighestAttestation(ctx, att); err != nil {
```

Comment:
you should also update highest on isslashble attestation endpoint

Reply:
gotcha

Reply:
you mean return highest there?

Reply:
update highest if it changes through the endpoint